CodingChallenges/CC_117_7_Segment_Display/CC_117_7_Segment_Display.pde

```diff
@@ -0,0 +1,51 @@
+int[] nums = {0x7E, 0x30, 0x6D, 0x79, 0x33, 0x5B, 0x5F, 0x70, 0x7F, 0x7B};
```

Comment:
Can you add the usual header stuff here? Check other coding challenges for an example? Thank you!!!

Reply:
I just added the usual header stuff. 
Thanks for all Dan.